src/levelEditor/sidebar/objectsList/SidebarObjectText.js

```diff
@@ -6,12 +6,14 @@ import type {GameObjectType} from '../../types/GameObjectType';
 import convertLevelIdToCoordinates from '../../util/convertLevelIdToCoordinates';
 
 type Props = $ReadOnly<{
-	obj: GameObjectType,
+	obj: ?GameObjectType,
```

Comment:
Not needed if SidebarEntityItem.js checks that the object is non-null
```suggestion
	obj: GameObjectType,
```